Gemfile

```diff
@@ -1,13 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'fastlane'
-gem 'hockeyapp'
 gem 'json'
 gem 'xcodeproj'
 
-# rubocop:disable Eval
```

Comment:
Good move.  I was wondering why it was necessary to load this 'Pluginfile' from the Gemfile, but it seems as though this is just Fastlane's way of tracking plugins and getting its own file to manage.  ([docs](https://docs.fastlane.tools/plugins/create-plugin/#add-a-plugin-to-your-project))

Reply:
Yeah, they have a "fastlane add_plugin" command that expects the Pluginfile, but I think it's just as easy to use the Gemfile.